zipship/rhel/roles/dev_env/molecule/default/converge.yml

```diff
@@ -4,6 +4,6 @@
   gather_facts: true
```

Comment:
Hey Jeff, I can't get a clean run here either 
Steps I ran:
-  molecule prepare
-  source ~/.bashrc
-  verified that oc worked in the terminal
-  molecule test

```
TASK [Create molecule instance as containers] **********************************
failed: [localhost] (item={'image': 'registry.redhat.io/rhel8/python-39', 'name': 'molecule-rhel8-python-39', 'workingDir': '/tmp'}) => {"ansible_loop_var": "item", "changed": false, "item": {"image": "registry.redhat.io/rhel8/python-39", "name": "molecule-rhel8-python-39", "workingDir": "/tmp"}, "msg": "Failed to import the required Python library (kubernetes) on workspacebcd999b20b70450d-5646cc7fb4-htzc4's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}
failed: [localhost] (item={'image': 'registry.redhat.io/rhel8/python-38', 'name': 'molecule-rhel8-python-38', 'workingDir': '/tmp'}) => {"ansible_loop_var": "item", "changed": false, "item": {"image": "registry.redhat.io/rhel8/python-38", "name": "molecule-rhel8-python-38", "workingDir": "/tmp"}, "msg": "Failed to import the required Python library (kubernetes) on workspacebcd999b20b70450d-5646cc7fb4-htzc4's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}

PLAY RECAP *********************************************************************
localhost                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```